resources/nl-core/nl-core-NameInformation.xml

```diff
@@ -35,7 +33,100 @@
       <alias value="nl-core-NameInformation" />
+      <constraint>
+        <key value="nl-core-NameInformation-01" />
+        <severity value="warning" />
+        <human value="If parts of the name are specified using one or more of the structured elements, the `.text` element SHOULD be populated with a rendered version of the full name. This ensures compatibility with common FHIR usage." />
+        <expression value="(family.exists() or given.extension.where(url = 'http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier').where(value != 'IN').exists()) implies text.hasValue()" />
+      </constraint>
+    </element>
+    <element id="HumanName.text">
+      <path value="HumanName.text" />
+      <comment value="A representation of the full name, including given names and the familiy name. Although this element represents the official names, it is considered desirable to include unofficial names as well if commonly used, for example added in brackets.&#xD;&#xA;&#xD;&#xA;When implementing zib NameInformation, this rendition should include GivenName or Initials, the GivenName and the family name constructed according to the NameUsage concept." />
+      <condition value="nl-core-NameInformation-01" />
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.4" />
+        <comment value="FirstNames (implicit, main mapping is on .given)" />

```

Comment:
Cosmetic note: I noticed in the whole repo that sometimes the [element name] in the implicit part
 is given based on the full path and sometimes with the dot notation. We should harmonize this. I prefer this relative path.

Reply:
Could be a todo on our board.

Reply:
Created a low-priotiry task on our board.